pytest.ini

```diff
@@ -10,3 +10,5 @@ markers =
     apps_to_insert: specify applications to seed the db
     unique_fund_round: use a unique fund and round (randomly generated ids) for each test
     fund_round_config: specify funds and rounds to use for tests
+    function_calls_to_mock:
```

Comment:
might it be useful to have descriptors here as with other markers?

Reply:
Yep! Done